### ISSUE TITLE
Video not playing on Multiscreen

### ISSUE THREAD

**Issue by DLarner** [OrangeJedi/Aerial#110]
**Describe the bug**
On the Multiscreen settings, both options are not ticked to allow different videos to play on different monitors. The videos show as stills but do not play. If I tick the option "Only Play videos on the primary screen" then the video plays fine.

**To Reproduce**
This happens when launched after 10 mins or by right clicking the icon on the taskbar and clicking Start Aerialy

**Expected behavior**
I expect the videos to play

**Screenshots**
N/A

**Aerial Version**
Happened on all version including latest V1.0.2

**Additional context**

**Comment by OrangeJedi** [comment]
Just to be clear:
1) Neither option under the multi-screen menu is checked
2) The videos play completely fine if  `Only play videos on the primary screen` is checked
&
3) This is happening consistently

And you say that Aerial plays fine for a few minutes then stops? Can you make it stop faster by cycling videos with the arrow key?

**Comment by DLarner** [comment]
Points 1-3 are correct. However, the last sentence in incorrect as the videos don’t play at all on multi-screen.

**Comment by OrangeJedi** [comment]
Ok, that makes more sense. I take it the preview from the configuration screen works okay?

I'll take a look in the code and see if I can find what might be causing this, though I have no ideas off the top of my head.

Because you are seeing stills from the videos but they aren't playing it could be an issue with performance. Do you have any other apps running in the background? What are the specs of your computer?

**Comment by DLarner** [comment]
Yes, preview works fine, albeit in a window, not full screen.

Not many apps running in background and got a high spec laptop - Dell XPS 17 with 32Gb memory. Was running the previous BETA versions of Aerial fine.

**Comment by Alexyoe** [comment]
I'm having the same issue. I kept Task Manager on top and there are no performance issues happening. Does Aerial output a log file anywhere?

**Comment by Alexyoe** [comment]
I think I've found the source of the issue. It seems to have something to do with the style section. Disabling all video processing allows the videos to play but loses the nice fade in.

**Comment by OrangeJedi** [comment]
This is all so weird. Both screen use the exact same code (and the same code as when you have just one screen on) so I have very little idea on what it could be. At least now we know it isn't anything with communication between scripts or internet connection.

Thanks @Alexyoe for testing it some more. I can't seem to replicate the issue myself so that's super helpful. Right now Aerial doesn't have a very robust logging feature, but you might be able to open the devtools by starting Aerial from the command line by running `Aerial.exe /s /nq` and then pressing `control-shift-i`. You'll have to close the app with task manager or `alt-f4` though.

If the disable video processing is making it work then it sounds like it might be an issue with how the video is processing. I'll take a closer look at that code the next time I can.

**Comment by Alexyoe** [comment]
Thanks for the suggestion @OrangeJedi. I tried using `Aerial.exe /s /nq` but when I hit `control-shift-i` it kills the screensaver. I've tried to setup a dev environment to see if I can help diagnose the issue but  I keep getting errors when running `npm install`. What version of Node should I be using for the dev environment?

**Comment by OrangeJedi** [comment]
So I checked the code for the `/nq` 'no-quit' option and it turns out that it was pretty much disabled in the last update to fix a bug. I added a commit to the [v1.1.0 branch](https://github.com/OrangeJedi/Aerial/tree/v1.1.0) to enable debugging with it.

As for building the dev environment I'm using Node v16.17.1 & NPM v8.19.2. Hopefully the versions help. What kind of errors were you getting from `npm install`?

**Comment by Alexyoe** [comment]
I was able to get the 1.1.0 branch installed by deleting the `package-lock.json` before running `npm install`. It looks like the `/nq` and `/dt` flags aren't working for me. 

I tried adding some comments in the `videoQuality` section but can't seem to get any output. I'm guessing it is logging to the electron console and not my own?

**Comment by Alexyoe** [comment]
I was able to get the dev console open while running with `/s` and `/nq`. The only thing I'm seeing is some warning about source maps. 

Weirdly, when opening and closing the dev console the video moves forward slightly and if I close one of the two Aerial windows the other plays at high speed to catch up to where it should be and then plays at normal speed.

Another thing I've noticed is when running with just `/s` the videos sometimes plays for a fraction of a second before freezing. I'll keep poking around and see if I can't find where the issue is stemming from.
